src/v2/BaseLoginRouter.js

```diff
@@ -185,6 +186,18 @@ export default Router.extend({
     // }
   },
 
+  handleIdxInteractionCanceled() {
+    // Restart login flow
+    this.settings.set('useInteractionCodeFlow', true);
+    startLoginFlow(this.settings)
```

Comment:
this creates duplication and alienate `interact` from main flow, hard to maintain widget state properly in case of errors..